content/applications/finance/accounting/receivables/customer_payments/batch.rst

```diff
@@ -2,34 +2,20 @@
 Batch payments: Batch deposits (checks, cash etc.)
 ==================================================
 
-A **Batch Deposit** groups multiple payments in a single batch. This allows you to deposit several
-payments into your bank account with a single transaction. This is particularly useful to deposit
-cash and checks.
+A **Batch Deposit** groups multiple payments in a single batch. This allows depositing several
+payments into your bank account with a single action. It can be advantageous to deposit any cash and
+check transactions rapidly.
 
-This feature allows you to list several customer payments and print a **deposit slip**. This ticket
-contains the details of the transactions and a reference to the batch deposit. You can then select
-this reference during a bank reconciliation to match the single bank statement line with all the
-transactions listed in the batch deposit.
+This feature lets you list several customer payments and print a deposit slip. It contains the
+details of the transactions and a reference to the batch deposit. You can then select this reference
+during a bank reconciliation to match the single bank statement line with all the transactions
+listed in the batch deposit.
```

Comment:
I would simplify. Proposal:
A **batch deposit** is a convenient way to group multiple payments and deposit them into your bank account simultaneously. It saves time and effort, especially when dealing with cash and checks. With this feature, you can list multiple customer payments, generate a detailed deposit slip with a batch reference, and use this reference during bank reconciliation to easily match your bank statement lines with the transactions in the batch deposit.